x-pack/plugin/esql/qa/testFixtures/src/main/resources/ip.csv-spec

```diff
@@ -485,3 +485,52 @@ beta         | 127.0.0.1
 beta         | 127.0.0.1
 beta         | 127.0.0.1
 ;
+
+ipPrefix
+//tag::ipPrefix[]
+required_capability: fn_ip_prefix
+row ip4 = to_ip("1.2.3.4"), ip6 = to_ip("fe80::cae2:65ff:fece:feb9")
+| eval ip4_prefix = ip_prefix(ip4, 24), ip4_as_6_prefix = ip_prefix(ip4, 120), ip6_prefix = ip_prefix(ip6, 112), ip6_as_4_prefix = ip_prefix(ip6, 16),
```

Comment:
This is probably too many to fit in an example. Maybe make a couple of csv tests with the examples? It's fine to do them all at once in a test, but if we're stealing them for example for users let's just give them in individual examples. Sorry it's more typing!

Reply:
Makes sense, I wasn't sure, and tbh forgot to check the docs (Which are still a 404 in the PR link, but whatever).

Gonna divide them by usecase, keeping a "simple" one as the doc